production.ini

```diff
@@ -303,6 +303,9 @@ use = egg:bodhi-server
 # want to use 'koji'.
 # buildsystem = dev
 
+# Koji's web URL
```

Comment:
Let's give this more of a description than this - this comment is the exact same text as the name of the variable.